pkg/config/config.go

```diff
@@ -1310,15 +1321,20 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 			logrus.Infof("Checkpoint/restore support disabled via configuration")
 		}
 
-		if err := c.seccompConfig.LoadProfile(c.SeccompProfile); err != nil {
+		if c.SeccompProfile == "" {
+			if err := c.seccompConfig.LoadDefaultProfile(); err != nil {
+				return fmt.Errorf("unable to load default seccomp profile: %w", err)
+			}
+		} else if err := c.seccompConfig.LoadProfile(c.SeccompProfile); err != nil {
 			if !errors.Is(err, os.ErrNotExist) {
 				return fmt.Errorf("unable to load seccomp profile: %w", err)
 			}
 
-			logrus.Info("Specified profile does not exist on disk")
+			// Fallback to the internal default in order not to not break upgrade paths.
+			logrus.Info("Seccomp profile does not exist on disk, fallback to internal default profile")
```

Comment:
Is this typo? The comment has duplicate "not". same for other places having this comment.

btw, @saschagrunert do you know why does it fall back to the default?
It seems introduced in this PR. https://github.com/cri-o/cri-o/pull/6029
I think it should raise an error, otherwise there's a risk of having a different seccomp profile without noticing it.